client/src/interface/Column.ts

```diff
@@ -0,0 +1,8 @@
+export interface Column {
+  Id: string;
+  Title: string;
+  Tasks: string[];
```

Comment:
In JavaScript, it is convention for camelCase. Might be best to follow this from now on 👍🏻 😄